package.json

```diff
@@ -224,10 +224,10 @@
 		"build-docker": "node bin/build-docker.js",
 		"build-static": "ncp static public",
 		"prebuild-server": "mkdirp build",
-		"build-server": "cross-env-shell BROWSERSLIST_ENV=server NODE_PATH=$NODE_PATH:server:client:. webpack --display errors-only --config webpack.config.node.js",
+		"build-server": "cross-env-shell BROWSERSLIST_ENV=server NODE_PATH=$NODE_PATH:server:client:. calypso-build --display errors-only --config webpack.config.node.js",
 		"build-client": "npm run build-client-evergreen",
-		"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --display errors-only",
-		"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --display errors-only",
+		"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./packages/calypso-build/bin/calypso-build.js --display errors-only",
+		"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./packages/calypso-build/bin/calypso-build.js --display errors-only",
```

Comment:
The `node $NODE_ARGS ./packages/calypso-build/bin/calypso-build.js` stuff is mildly annoying but apparently required to take ` NODE_ARGS=--max_old_space_size=8192` from [L232](https://github.com/Automattic/wp-calypso/pull/33035/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R232) into account. Ideally, we can figure out why building Calypso in production mode takes up so much memory, fix the issue, and drop this stuff.